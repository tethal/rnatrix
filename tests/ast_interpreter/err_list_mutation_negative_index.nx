fun main() {
    var list = [1, 2, 3];
    list[-1] = 99;
}