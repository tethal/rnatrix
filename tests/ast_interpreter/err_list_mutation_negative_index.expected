<string>:3:5: error: index cannot be negative
    list[-1] = 99;
    ^^^^^^^^
